docs/content/en/integrations/importing.md

```diff
@@ -118,4 +118,23 @@ A classic way of reimporting a scan is by specifying the ID of the test instead:
     "scan_type": 'ZAP Scan',
     "test": 123,
 }
-```
\ No newline at end of file
+```
+
+## Using the `scan_date` field
```

Comment:
Is this only about `scan_date` in the API or about **Scan Completion Date** in the UI as well?